packages/office-ui-fabric-react/src/utilities/decorators/withViewport.tsx

```diff
@@ -116,12 +116,11 @@ export function withViewport<TProps extends { viewport?: IViewport }, TState>(
 
     public render(): JSX.Element {
       const { viewport } = this.state;
-      const { skipViewportMeasures } = this.props as IWithViewportProps;
-      const isViewportVisible = skipViewportMeasures || (viewport!.width > 0 && viewport!.height > 0);
+      const newViewport = viewport!.width > 0 && viewport!.height > 0 ? viewport : undefined;
```

Comment:
Is checking `> 0` going to cause a cascading update due to having `minWidth` and `minHeight` set inline to `1` on line 122? Does the mount happen after all its children have been reconciled?

Reply:
It should not, the entire `withViewport` component triggers a cascading update because render state prior to this change was modified during React's reconciliation stage. That led React to re-reconcile the component and its children which included the static `DIV` you mention. The in-line styles are unrelated to that concern.

If that wrapper element was a separate component that mutated its state internally during render such as `width` and `height` then we would have double cascades (I think). Luckily it is a static HTML element today.